WasteWar/Assets/Scripts/Camera Controls/Move.cs

```diff
@@ -0,0 +1,40 @@
+﻿using UnityEngine;
+
+public class Move : MonoBehaviour
+{
+    [SerializeField]
+    private Terrain terrain;
+    private Vector3 terrainSize;
+    private Vector3 camPosition;
+    private Transform cam;
+
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        terrainSize = terrain.terrainData.size;
+
+        //setting cam to the center of terrain
```

Comment:
pull this code into function, call it centerCamera() and remove this comment aftwards